chat_shell/chat_shell/services/chat_service.py

```
@@ -288,6 +288,31 @@ async def _process_chat(
+            if request.enable_deep_thinking and request.user_subtask_id:
+                last_msg_content = messages[-1].get("content") if messages else None
+                if isinstance(last_msg_content, list):
+                    try:
+                        from chat_shell.history import update_user_message_content
+
+                        await update_user_message_content(
+                            task_id=request.task_id,
+                            user_subtask_id=request.user_subtask_id,
+                            content=last_msg_content,
+                        )
+                    except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)